editor/connections_dialog.cpp

```diff
@@ -488,6 +489,34 @@ Vector<Variant> ConnectDialog::get_binds() const {
 	return cdbinds->params;
 }
 
+String ConnectDialog::get_signature(const MethodInfo &p_method, PackedStringArray *r_arg_names) {
+	PackedStringArray signature;
+	signature.append(p_method.name);
+	signature.append("(");
+
+	for (int i = 0; i < p_method.arguments.size(); i++) {
+		if (i > 0) {
+			signature.append(", ");
+		}
+
+		const PropertyInfo &pi = p_method.arguments[i];
+		String tname = "var";
+		if (pi.type == Variant::OBJECT && pi.class_name != StringName()) {
+			tname = pi.class_name.operator String();
+		} else if (pi.type != Variant::NIL) {
+			tname = Variant::get_type_name(pi.type);
+		}
+
+		signature.append((pi.name.is_empty() ? String("arg " + itos(i)) : pi.name) + ": " + tname);
+		if (r_arg_names) {
+			r_arg_names->push_back(pi.name + ":" + tname);
```

Comment:
Missing space before type name.

Reply:
It was already like that. It's required in this format for `script_add_function_request()`.